data_models/environments/dev

```diff
@@ -0,0 +1,8 @@
+DJANGO_SECRET_KEY=!f3p74+hi5omz+)m((+8597&&01h-&bf(0#d1mwvak&clmv*%g
```

Comment:
Since this secret is only used for development right now, it is okay with me to have it exposed. Maybe you can add an issue to remind yourself this fact. 

On the other hand, since it is exposed to the rest of the world, can we simply use something as simple as "foobar"?